packages/next/src/export/index.ts

```diff
@@ -714,23 +723,22 @@ export default async function exportApp(
             errorPaths.push(page !== path ? `${page}: ${path}` : path)
           }
 
-          if (options.buildExport && configuration) {
+          if (options.buildExport) {
```

Comment:
I think this might have been a bug before. 

Previously, it only read `nextConfig` if it was passed in as an arg, but not the case when it read the file on line 181 above